common/persistence/cassandraPersistence_test.go

```diff
@@ -1754,3 +1943,21 @@ func copyReplicationInfo(sourceInfo *ReplicationInfo) *ReplicationInfo {
 		LastEventID: sourceInfo.LastEventID,
 	}
 }
+
+func serializeHistoryEvents(events []*gen.HistoryEvent) *SerializedHistoryEventBatch {
```

Comment:
can you add (s *cassandraPersistenceSuite)  so this function will not be used by src code?

Reply:
Done.